sycl/include/sycl/ext/intel/experimental/esimd/common.hpp

```diff
@@ -57,15 +57,6 @@
 
 // Macros for internal use
 #define __ESIMD_NS sycl::ext::intel::experimental::esimd
-#define __ESIMD_QUOTE1(m) #m
-#define __ESIMD_QUOTE(m) __ESIMD_QUOTE1(m)
-#define __ESIMD_NS_QUOTED __ESIMD_QUOTE(__ESIMD_NS)
-#define __ESIMD_DEPRECATED(new_api)                                            \
-  __SYCL_DEPRECATED("use " __ESIMD_NS_QUOTED "::" __ESIMD_QUOTE(new_api))
```

Comment:
I suggest to leave this one and above in place, as we'll likely have to deprecate APIs in future.

Reply:
Ok.